perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -2318,4 +2328,21 @@ private static String getUserPreferredEmail(PerunSession sess) {
 		}
 	}
 
+	/**
+	 * Returns configuration's brand in which the provided domain is specified
+	 * @param domain
+	 * @return brand or null if domain is not present in the configuration
+	 */
+	private static PerunAppsConfig.Brand getBrandContainingDomain(String domain) {
```

Comment:
This kind of search function could be part of `PerunAppsConfig` itself, what do you think @Vojtech-Sassmann ?

Reply:
Yeah, definitely.

Reply:
Okay, moved to `PerunAppsConfig`.